### PR TITLE
Add policies

### PR DESCRIPTION
**Pull request by pkattera**
# Description

Adds new policy pages for Terms of Use, Cookies Policy, Privacy Policy and links to them from the Connect Wallet dialog.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other

## How should this be tested?

Please provide instructions so we can test. Please also list any relevant details for your test configuration.

- [ ] Click on the policy links from the Connect a Wallet screen.
- [ ] Test on light mode and dark mode
- [ ] Test on different screen sizes

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code where relevant, particularly in hard-to-understand areas
- [ ] My changes generate no new console warnings
- [ ] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/pages/cookies-policy.vue [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'cookies-policy'
+};
+</script>
+
+<style scoped></style>
```
Comment (garethfuller) [inline src/pages/cookies-policy.vue:112]: We can remove this.

File: src/components/web3/WalletSelectModal.vue [hunk 2]
```diff
@@ -4,16 +4,40 @@
     @close="$emit('close')"
     title="Connect to a wallet"
   >
+    <p class="pb-3 text-sm">
+      By connecting a wallet, I agree to Balancer Labs’
```
Comment (garethfuller) [inline src/components/web3/WalletSelectModal.vue:8]: Let's move the text in this component to the default.json locales file.

**Review by garethfuller — APPROVED** [review]
Looks good, just a few minor comments.

File: src/components/web3/WalletSelectModal.vue (hunk 2 above)
Comment (pkattera) [reply src/components/web3/WalletSelectModal.vue]: Added these, plus the 'New to Ethereum' section below to the default.json locales file

File: src/pages/cookies-policy.vue (hunk 1 above)
Comment (pkattera) [reply src/pages/cookies-policy.vue]: Removed these

**Review by garethfuller — APPROVED** [review]

**Pull request merged**